lib/data_types/error.dart

```diff
@@ -0,0 +1,19 @@
+enum TrackingError {
```

Comment:
@pavel-kuznetsov-hypertrack why Tracking and not just HyperTrack.Error?

Reply:
Inner classes is not possible in Dart